DESCRIPTION

```diff
@@ -49,7 +49,7 @@ Suggests: bit64, curl, R.utils, knitr, xts, nanotime, zoo, yaml
 SystemRequirements: zlib
 Description: Fast aggregation of large data (e.g. 100GB in RAM), fast ordered joins, fast add/modify/delete of columns by group using no copies at all, list columns, friendly and fast character-separated-value read/write. Offers a natural and flexible syntax, for faster development.
 License: MPL-2.0 | file LICENSE
-URL: http://r-datatable.com
+URL: https://github.com/Rdatatable/data.table
```

Comment:
Why this change?

Reply:
I am not precisely sure, https://pkgdown.r-lib.org/articles/linking.html vignette mention to use github repo url. I reverted back to use two urls, as initially commited by Hadley

Reply:
I am not sure but we might need to use three urls there, as pkgdown website will be published under: https://Rdatatable.gitlab.io/data.table